angular-firebase/package.json

```diff
@@ -23,6 +23,7 @@
     "@ng-bootstrap/ng-bootstrap": "^6.0.0",
     "ethers": "^4.0.45",
     "firebase": "^7.9.3",
+    "ngx-toastr": "^12.0.0",
```

Comment:
If we have bootstrap implemented, we can first see what kind of options it provide to us:
https://getbootstrap.com/docs/4.3/components/alerts/
https://getbootstrap.com/docs/4.3/components/toasts/

Reply:
I checked for bootstrap toast but I think ngx-toaster is better.

Reply:
I have looked at the bootstrap toast but I think ngx-toaster is better.